llvm/lib/Target/AMDGPU/AMDGPUMIRFormatter.cpp

```diff
@@ -17,6 +17,157 @@
 
 using namespace llvm;
 
+void AMDGPUMIRFormatter::printImm(raw_ostream &OS, const MachineInstr &MI,
+                      std::optional<unsigned int> OpIdx, int64_t Imm) const {
+
+  switch(MI.getOpcode()) {
```

Comment:
I don't understand why the automated clang-format checker isn't complaining about things like this. When I run clang-format locally it does complain.
```suggestion
  switch (MI.getOpcode()) {
```